docs/workforce-management-framework/topics/con-security-introduction.adoc

```diff
@@ -0,0 +1,49 @@
+[]
+= Introduction to {WFM-RC-NameShort} security features
+
+{WFM-RC-NameShort} server gives developers a choice one of the following authentication and authorisation providers
+link:http://www.keycloak.org/[Keycloak] and link:http://passportjs.org/[Passport.js].
+{WFM-RC-NameShort} is build around security interface, which abstracts from any security libraries.
+Both implementations can be swapped at any time without breaking internal api of {WFM-RC-NameShort} modules and reference applications.
+Users can also provide their own security implementation by implementing their own security interface.s
+
+Another security measures are incorporated into our reference applications.
```

Comment:
"Another security measures are incorporated into our reference applications." - I don't see any value in what this sentence provides, i think it can be removed.

Reply:
Should Read Other, to cover any future improvements like security pin.,